python/openmldb_tool/diagnostic_tool/diagnose.py

```diff
@@ -127,22 +128,65 @@ def inspect_offline(args):
     jobs = conn.execfetch("SHOW JOBS")
     # TODO some failed jobs are known, what if we want skip them?
     print(f"inspect {len(jobs)} offline jobs")
-    fails = []
     # jobs sorted by id
     jobs.sort(key=lambda x: x[0])
-    # only FINAL_STATE "finished", "failed", "killed", "lost"
-    final_failed = ["failed", "killed", "lost"]
-    for row in jobs:
-        if row[2].lower() in final_failed:
-            fails.append(" ".join([str(x) for x in row]))
-            # DO NOT try to print rs in execfetch, it's too long
-            std_output = conn.execfetch(f"SHOW JOBLOG {row[0]}")
-            # log rs schema is FORMAT_STRING_KEY
-            assert len(std_output) == 1 and len(std_output[0]) == 1
-            print(f"{row[0]}-{row[1]} failed, job log:\n{std_output[0][0]}")
-    fails_total = "\n".join(fails)
-    assert not fails, f"failed jobs:\n{fails_total}"
-    print("all offline final jobs are finished")
+    job_state = getattr(args, "state", "failed")
+    jobs_num, jobs_show = get_state_jobs(job_state, jobs)
+    print(f"{jobs_num} {job_state} jobs")
+    print(*jobs_show, sep="\n")
+
+
+def get_state_jobs(state: str, jobs):
+    job_state_field = {
+        "running": ["running"],
+        "finished": ["finished"],
+        "failed": ["failed", "killed", "lost"]
```

Comment:
what if stop job when job is running or submitted? the stoped job state is what?